src/components/Warnings/StETHCollateralWarning.tsx

```diff
@@ -0,0 +1,10 @@
+import { Trans } from '@lingui/macro';
+
+export const StETHCollateralWarning = () => {
+  return (
+    <Trans>
+      Due to stETH contracts limitations stETH supply position cannot be swapped to another asset.
```

Comment:
Due to stETH <Link href="https://github.com/lidofinance/lido-dao/issues/442">contract limitation</Link>, stETH position cannot be utilized with repay with collateral or collateral swap. astETH token may be available to swap on exchanges.

Reply:
About this, you can use collateral swap if you're going from other asset to `stETH`, what you can't do is the reverse.

Reply:
Makes sense, in that case 'or collateral swap' could be 'or swap collateral as source token'

Reply:
Still think this should be reworded, and maybe reference [this doc](https://docs.lido.fi/token-guides/steth-on-aave-caveats/) instead of just calling it a contract limitation

Reply:
Mmm, hard to think what it should say. 

"Due to a contract limitation of stETH (link to doc), it is not possible to perform a collateral swap where stETH is the base asset."?

Reply:
"Due to internal stETH mechanics required for rebasing support, it is not possible to perform a collateral swap where stETH is the source token" ?

With no link to doc? I'm not sure if it'd be more or less confusing for people after reading through the caveats.